src/Microsoft.TestPlatform.CrossPlatEngine/Adapter/FrameworkHandle.cs

```diff
@@ -110,6 +110,20 @@ public int LaunchProcessWithDebuggerAttached(string filePath, string workingDire
             return this.testRunEventsHandler.LaunchProcessWithDebuggerAttached(processInfo);
         }
 
+        /// <inheritdoc />
+        public bool AttachDebuggerToProcess(int pid)
+        {
+            if (pid < 0)
```

Comment:
it it also very unlikely that you will be able to attach to process 0 which is `Idle` on windows, and swapper on linux, I would at least write a warning into eqtrace if we get process id 0, that's probably an operator error. Same situation with the not supported exception wording and is as check

Reply:
Done.